website/client-old/js/controllers/groupsCtrl.js

```diff
@@ -2,6 +2,7 @@
 
 habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '$http', '$q', 'User', 'Members', '$state', 'Notification',
   function($scope, $rootScope, Shared, Groups, $http, $q, User, Members, $state, Notification) {
+    $scope.inviteOrStartParty = Groups.inviteOrStartParty;
```

Comment:
You changed the function name here, which will cause an issue. openInviteModal is different than inviteOrStartParty although they share much logic.